### ISSUE TITLE
Ask authors to upload export of ref manager library, pref in bibtex format?

### ISSUE THREAD

**Issue by DeckOfPandas** [NSAMR/uk.ac.nsamr.journal#406]
This would be a single button click in JabRef, a small poke in Mendeley, single button click in EndNote etc

**Comment by srees95** [comment]
i'm not sure all reference managers will export in the same format, main ones are: refworks, the one built into word (gah), mendely

**Comment by DeckOfPandas** [comment]
Yes, but some export formats are more generic than others and can be imported far more easily to new places, and most will have those even if they don't have fancier ones.

Mendeley has bibTeX export built in
As does RefWorks
As does Zotero
It's a free addon for EndNote

**Comment by DeckOfPandas** [comment]
Word can import a bibTeX export, which is a good idea anyway -- use a proper ref manager, export to .bib, import to word if you must

Etc

**Comment by DeckOfPandas** [comment]
https://libguides.usask.ca/c.php?g=218034&p=1446316
